spec/unit/blockscore/collection/member_spec.rb

```diff
@@ -1,7 +1,32 @@
 module BlockScore
   class Collection
     RSpec.describe Member do
-      describe '.save' do
+      describe '#save' do
+        let(:question_set) { QuestionSet.new }
+        subject(:member) { described_class.new(person, question_set) }
+
+        context 'unsaved person' do
+          let(:person) { BlockScore::Person.new(attributes_for(:person)) }
+
+          it 'saves the person as part of member save' do
+            expect(person).to_not be_persisted
+            expect(member.save).to be true
+            expect(person).to be_persisted
+          end
+        end
+
+        context 'previously saved person' do
+          let(:person) { BlockScore::Person.create(attributes_for(:person)) }
+
+          it 'contains the proper fields' do
```

Comment:
This example tests attributes individually. Resutling in the potential miss of specification when attributes are added in future.

What about to (assuming correct equalization exists) test against an expected instance. (+ the test of persistence as this is likely not part of the equalization behavior).

This would also eliminate the `be_an_instance_of` instance below.

Commenting just seeing the specs, assuming behavior of the class.